content/doc/book/scaling/architecting-for-scale.adoc

```diff
@@ -96,6 +96,7 @@ possible to horizontally scale an architecture, which allows a single Jenkins
 installation to host a large number of projects and build environments.
 
 [[master/agent-communication-protocols]]
```

Comment:
@timja is correct.  This was an attempt to retain an anchor that had existed in the original document and included a `/` in the anchor definition.  I can't find any way in asciidoc to define an anchor that includes a `/`.  Let's delete the anchor.
```suggestion
```